artificial-scrum-master/apps/artificial.scrum.master.frontend/src/app/features/sprint-stats/components/stats/stats.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, computed, input } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { GetSprintStats } from '../../models/GetSprintStats';
+import { MaterialModule } from '../../../../shared/material.module';
+import { TranslateModule } from '@ngx-translate/core';
+
+@Component({
+  selector: 'app-stats',
+  standalone: true,
+  imports: [CommonModule, MaterialModule, TranslateModule],
+  templateUrl: './stats.component.html',
+})
+export class StatsComponent {
+  stats = input.required<GetSprintStats>();
+
+  public totalPoints = computed(() => {
+    console.log('totalPoints computed');
+    if (!this.stats().totalRolePoints) {
+      return -1;
```

Comment:
magic number? weź sobie te numerki z tej klasy do jakiś private readonly fieldów w tej klasie i daj im jakieś nazwy